src/app/core/entity-details/related-time-period-entities/time-period.ts

```diff
@@ -26,6 +26,7 @@ export class TimePeriod extends Entity {
    * Returns true if this relation is currently active
    */
   get isActive(): boolean {
+    // TODO: isActive time period vs. "archived" record?
```

Comment:
I have the feeling we might need to split these two concepts

Reply:
_as discussed just now:_

we will keep the overlapping logic with special isActive functions of Todo and TimePeriodEntity and consider these as not active similar to the manual archiving.